### PR TITLE
Bug fixes 1907

### PR DESCRIPTION
**Pull request by TarunKrShukla**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ARMChecker.Lib/ControlDataMatchType.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@ public enum ControlDataMatchType
 		NotAllow,
 		GreaterThan,
 		LesserThan,
-		Equals
-	}
+		Equals,
+        GreaterThanEqual,
```
Comment (VishalHaibatpure) [inline src/AzSK.ARMChecker.Lib/ControlDataMatchType.cs]: Can you discuss this word with Manish. Should it be GreaterThan**Or**Equal

File: src/AzSK.ARMChecker.Lib/ControlEvaluator.cs [hunk 2]
```diff
@@ -190,6 +192,36 @@ private static ControlResult EvaluateStringSingleToken(ResourceControl control,
             return result;
         }
 
+        private static ControlResult EvaluateSingleVersionToken(ResourceControl control, JObject resource)
```
Comment (VishalHaibatpure) [inline src/AzSK.ARMChecker.Lib/ControlEvaluator.cs]: Code comments missings

File: src/AzSK.ARMChecker.Lib/ControlDataMatchType.cs (hunk 1 above)
Comment (TarunKrShukla) [reply src/AzSK.ARMChecker.Lib/ControlDataMatchType.cs]: Ok i will discuss

File: src/AzSK.ARMChecker.Lib/ControlEvaluator.cs (hunk 2 above)
Comment (TarunKrShukla) [reply src/AzSK.ARMChecker.Lib/ControlEvaluator.cs]: ??

**Pull request merged**